frontend/src/components/application/Details.tsx

```diff
@@ -214,10 +233,28 @@ const Details: FunctionComponent<Props> = ({
           <div>
             <h3 className="text-xl">{app.summary}</h3>
             <div
-              className={`prose dark:prose-invert xl:max-w-[75%]`}
-              dangerouslySetInnerHTML={{ __html: description }}
+              {...getCollapseProps()}
+              className={`prose dark:prose-invert xl:max-w-[100%] ${
+                !isExpanded && scrollHeight > collapsedHeight
+              }`}
+              ref={ref}
+              dangerouslySetInnerHTML={{
+                __html: description,
+              }}
             />
           </div>
+          {scrollHeight > collapsedHeight && (
+            <button
+              className="w-full rounded-tl-none rounded-tr-none rounded-bl-none rounded-br-none border-t py-3 px-0 font-semibold hover:cursor-pointer hover:bg-colorHighlight dark:border-zinc-600"
+              {...getToggleProps()}
+            >
+              {isExpanded ? (
+                <span>{t("Se less")}</span>
```

Comment:
Translation keys should be lowercase with no spaces.

This one also has a typo